cli.js

```diff
@@ -1,60 +1,64 @@
 #!/usr/bin/env node
 
-var pipboylib = require('pipboylib')
-var relay = require('pipboylib/lib/relay')
+var pipboylib = require('pipboylib');
+var relay = require('pipboylib/lib/relay');
```

Comment:
By the way, obama is all semistandard under the covers. I made it in frustration of dueling linter wars.